app/code/Magento/Sales/etc/adminhtml/system.xml

```diff
@@ -132,6 +132,14 @@
                     <source_model>Magento\Config\Model\Config\Source\Enabledisable</source_model>
                     <backend_model>Magento\Sales\Model\Config\Backend\Email\AsyncSending</backend_model>
                 </field>
+                <field id="sending_limit" translate="label" type="text" sortOrder="2" showInDefault="1" showInWebsite="0" showInStore="0" canRestore="1">
+                    <label>Limit to sending emails</label>
```

Comment:
You're not limiting email sending, you're limiting how many entities (orders/shipments/etc) will be processed per one cron run.
Please Update label and description